daemon/api_users.go

```diff
@@ -341,196 +313,33 @@ func createUser(c *Command, createData postUserCreateData) Response {
 		createData.Sudoer = true
```

Comment:
this path arriving here was tested on master but is not anymore here, so we lost too much testing

Reply:
Re-added the removed tests which tested this